sdk/src/main/java/io/radar/sdk/RadarUtils.kt

```diff
@@ -88,6 +89,18 @@ internal object RadarUtils {
         return locationAuthorization
     }
 
+    internal fun getLocationAccuracyAuthorization(context: Context): String {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
This follows the pattern in iOS here:

https://github.com/radarlabs/radar-sdk-ios/blob/master/RadarSDK/RadarUtils.m#L90

Technically, though, we could simplify this implementation for Android and not do a `Build.VERSION_CODES.S` check, since `ACCESS_COARSE_LOCATION` has been around since Android version 1.

The benefit of the current approach is that it matches what iOS does.  OTOH, we could keep `compileSdkVersion: 29` if we didn't need to check `Build.VERSION_CODES.S` here.

Reply:
we're probably going to cut to a new major version with android 12, so it's probably fine to use this. I refactored this to do the `return if` style.